internal/image/live.go

```diff
@@ -79,6 +79,9 @@ func (img *LiveImage) InstantiateManifest(m *manifest.Manifest,
 		artifactPipeline = vmdkPipeline
 		artifact = vmdkPipeline.Export()
 	case platform.FORMAT_GCE:
+		// NOTE(akoutsou): temporary workaround; filename required for GCP
+		// TODO: define internal raw filename on image type
+		imagePipeline.Filename = "disk.raw"
```

Comment:
Yeah, the filename really matters. Requirements are defined in https://cloud.google.com/compute/docs/import/import-existing-image#requirements_for_the_image_file. You probably plan it anyway, but I suggest to merge this fix and the regenerated GCE manifests with the previous commit that changes the pipeline used for GCE images and the commit which regenerated GCE manifests (It contained a commit message that the filename does not matter, which would be confusing (because it does).

Reply:
yeah thanks.  I knew GCP had strict requirements for the archive but forgot that the internal filename was one of them until I saw the test failing.

Rebased and squashed.